person.rb

```diff
@@ -0,0 +1,20 @@
+class Person
+  attr_accessor :id, :name, :age
+
+  def initialize(id, age, name = 'Unknown', parent_permission: true)
+    @id = id
```

Comment:
You should randomly generate the id so that it will be unique to each person instead of making the user enter his/her own id which is not a good user experience 😄 .

You can consider using : `@id = Random.rand(1..1000)`